File: WEEK1/DAY1/Prog5/P5.py
a=int(input("Enter the first number: "))
b=int(input("Enter the second number: "))
c=int(input("Enter the third number: "))

if (a>=b) and (a>=c):
    ans=a
elif (b>=a) and (b>=c):
    ans=b
else:
    ans=c

print("The largest number is ",ans)

File: WEEK1/DAY1/Prog5/P5.c
#include <stdio.h>

int main(){
    int a,b,c,ans;
    printf("Enter the first number: ");
    scanf("%d",&a);
    printf("Enter the second number: ");
    scanf("%d",&b);
    printf("Enter the third number: ");
    scanf("%d",&c);

    /*
    if (a>=b){
        if (a>=c){
            ans=a;
        }
    }
    else {
        if (b>=c){
            ans=b;
        }
        else{
            ans=c;
        }
    }
    

    if (a>=b && a>=c){
        ans=a;
    }
    else{
        if (b>=a && b>=c){
            ans=b;
        }
        else{
            ans=c;
        }
    }
    */

    if (a>=b && a>=c){
        ans=a;
    }
    else if(b>=a && b>=c){
        ans=b;
    }
    else{
        ans=c;
    }

    printf("The largest number is %d\n",ans);

    return 0;
}